RadixWallet/Features/ImportOlympiaLedgerAccountsAndFactorSources/ImportOlympiaLedgerAccountsAndFactorSources.swift

```diff
@@ -347,122 +353,19 @@ extension LedgerHardwareWalletFactorSource.DeviceModel {
 	}
 }
 
-// MARK: - NameLedgerAndDerivePublicKeys
-public struct NameLedgerAndDerivePublicKeys: Sendable, FeatureReducer {
```

Comment:
This was moved into its own file, and reduced to only take care of the `NameLedger` part. The `DerivePublicKeys` is handled inside here (`ImportOlympiaLedgerAccountsAndFactorSources`).